src/client.ts

```diff
@@ -2150,4 +2153,31 @@ export class StreamChat<
   deleteBlockList(name: string) {
     return this.delete<APIResponse>(`${this.baseURL}/blocklists/${name}`);
   }
+
+  exportChannels(request: Array<ExportChannelRequest>) {
+    const payload = { channels: request.map((v) => makeExportPayload(v)) };
+    return this.post<APIResponse & ExportChannelResponse>(
+      `${this.baseURL}/export_channels`,
+      payload,
+    );
+  }
+
+  exportChannel(request: ExportChannelRequest) {
+    return this.exportChannels([request]);
+  }
+
+  getExportChannelStatus(id: string) {
+    return this.get<APIResponse & ExportChannelStatusResponse>(
+      `${this.baseURL}/export_channels/${id}`,
+    );
+  }
+}
+
+function makeExportPayload(v: ExportChannelRequest) {
```

Comment:
i'd merge this into `client.exportChannels` since it's simple and not used anywhere else